frontend/src/metabase/parameters/utils/parameter-type.ts

```diff
@@ -10,3 +13,9 @@ export function isFieldFilterParameter(
   const type = getParameterType(parameter);
   return FIELD_FILTER_PARAMETER_TYPES.includes(type);
 }
+
+// TODO: maybe move to the dashboard component?
+export function isSingleOrMultiSelectable(parameter: Parameter): boolean {
+  const type = getParameterType(parameter);
+  return SINGLE_OR_MULTI_SELECTABLE_TYPES.includes(type);
+}
```

Comment:
I believe Dalton is moving stuff in metabase/parameters/utils to metabase-lib. So maybe it belongs there

Reply:
I believe not, at least not yet.
The current pass is about moving MBQL artifacts.